librocksdb_sys/crocksdb/c.cc

```diff
@@ -31,6 +31,13 @@
 #include "rocksdb/utilities/backupable_db.h"
 #include "rocksdb/utilities/debug.h"
 #include "rocksdb/write_batch.h"
+
+#include "db/column_family.h"
+#include "table/sst_file_writer_collectors.h"
```

Comment:
Do we have to expose these internal types?

Reply:
Yes, if we don't change the rocksdb code.